00-HTML-CSS-basics/Ex02/css/style.css

```diff
@@ -0,0 +1,90 @@
+html {
+	font-family:'Amatica SC', Times, serif;
+	font-size: 14px;
+	box-sizing: border-box;
+}
+header {
+	background-color: #f44259;
+	text-align: center;
+}
+nav {
+	background-color: #8f42f4;
+}
+aside {
+	background-color: #417df4;
+}
+footer {
+	background-color: #f4a641;
+	text-align: center;
+}
+.header {
+	font-size: 46px;
+	display: block;
+	position: relative;
+	margin: auto 2%;
+	border-radius: 10px;
+}
+.footer {
+	font-size: 25px;
+	border-radius: 10px;
+	display: block;
+	position: relative;
+	width: 96%;
+	height: auto;
+	bottom: 0;
+	margin: 1% 2%;
+	clear: both;
+}
+.content {
+	font-size: 20px;
+	background-color: #42f4c5;
+	display: inline-block;
+	position: relative;
+	max-width: 75%;
+	height: 100%;
+	border-radius: 10px;
+	margin: 0% 1%;
+	float: left;
+}
+.navigation {
+	font-size: 30px;
+	display: inline-block;
+	position: relative;
+	float: left;
+	max-width: 100%;
+	height: 100%;
+	border-radius: 10px;
+}
+.sidebar {
+	font-size: 25px;
+	display: inline-block;
+	position: relative;
+	height: 100%;
+	border-radius: 10px;
+	float: right;
+}
+div.midsect {
+	display: relative;
+	margin: auto 2%;
+	height: 600px;
+	float: left;
+}
+.content footer {
+	display: block;
+	position: absolute;
```

Comment:
Do not use `display:block` with `position: absolute`. All elements that are absolute, are automatically treated as display:block. Also, I recommend you to remove the float with clear fix here.